server/datastore/mysql/scripts.go

```diff
@@ -171,8 +171,7 @@ func (ds *Datastore) ListPendingHostScriptExecutions(ctx context.Context, hostID
     id,
     host_id,
     execution_id,
-    script_id,
-    script_contents
```

Comment:
Just to make sure: did the caller(s) use that script contents? E.g. is that expected to be part of an API response or something like that? If so we'd need to join with the `script_contents` table to get it, but if not that's a very good thing, as the returned data could get quite big.

Reply:
:tada: that's great news!